packages/rocketchat-lib/server/methods/filterBadWords.js

```diff
@@ -0,0 +1,25 @@
+
+var Filter = Npm.require('bad-words');
+    
+    RocketChat.callbacks.add( 'beforeSaveMessage' , function(message){
+        // console.log(message)
+        
+        if( RocketChat.settings.get('Message_AllowBadWordsFilter') )
+            {
+                
+                var badWordsList = RocketChat.settings.get('Message_BadWordsFilterList');
+                    // console.log(badWordsList)
+                    //Add words to the blacklist
+                    if(!!badWordsList && badWordsList.length)
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 10 space characters but found 20.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2432869137/issues/source?bid=3199634&fileBranchId=3240346#l13)